cmd/config/init/unique_rollapp_id.go

```diff
@@ -37,6 +38,11 @@ func IsRollappIDUnique(rollappID string, initConfig config.RollappConfig) (bool,
 func VerifyUniqueRollappID(rollappID string, initConfig config.RollappConfig) error {
 	isUniqueRollapp, err := IsRollappIDUnique(rollappID, initConfig)
 	if err != nil {
+		_, ok := err.(*url.Error)
```

Comment:
why u need to check for error type?
u will skip this check for local hub for any error